lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -43,6 +43,7 @@
 use function explode;
 use function in_array;
 use function is_subclass_of;
+use function method_exists;
```

Comment:
This looks no longer relevant but isn't caught by PHP_CodeSniffer for some reason.

Reply:
Fixed. There are several other broken imports in that file, btw. :-/